.github/workflows/e2e-nightly.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: rtCamp/action-slack-notify@ae4223259071871559b6e9d08b24a63d71b3f0c0
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_CHANNEL: tendermint-internal
+          SLACK_CHANNEL: lazyledger-core
```

Comment:
@marbar3778 besides adding a secret and a channel that does exist, this does not require any changes, right?

Reply:
Correct